src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -420,7 +425,9 @@ class DataCenterFormula private constructor(
             network = formula.overriddenNetwork
             databaseComputer = formula.databaseComputer
             databaseVolume = formula.databaseVolume
+            accessRequester = formula.accessRequester
             adminPasswordPlainText = formula.adminPasswordPlainText
+            skipStartupWait = formula.skipStartupWait
```

Comment:
This part is easy to miss. The code seem work without it, but will break when used with `InfrastructureFormula` and the option will be overwritten back to `false`.

Reply:
Yes, it kinda easy to forget about copying constructors